### DISCUSSION TITLE
Concatenated content

### DISCUSSION THREAD

**Discussion by kameshsundaram** [Budibase/budibase#1436] [Help] [answered]
I want to define a column data to be a concatenated data from 3 other columns. How do I do that?
Pl help.
Thanks

**Comment by mjashanks** [comment, marked as answer]
Hi There. You cannot currently do this, although we intend to implement this soon.

Could you use bindings in your front end to achieve what you need? e.g. 

`{{ Repeater.Markers.FirstName }} {{ Repeater.Markers.MiddleName }} {{ Repeater.Markers.LastName }}`

**Reply by mjashanks** [reply]
In our latest release, you can now add a formula column type.

![image](https://user-images.githubusercontent.com/3524181/120993518-80c17900-c77b-11eb-8bea-5144b76bf603.png)
